README.md

```diff
@@ -206,7 +214,50 @@ layeredWordCloud.build(1, wordFrequencies2);
 layeredWordCloud.writeToFile("output/layered_word_cloud.png");
 ```
 
-**Normalizers**
+Create a ParallelLayeredWordCloud using 4 distinct Rectangles.<br>
+Every Rectangle will be processed in a separate thread, thus minimizing build-time significantly
+
+```groovy
```

Comment:
I say this because people regularly message me that the README examples don't work since they copy/paste them :)